src/App/Main.jsx

```diff
@@ -28,8 +28,11 @@ import routes from './routes';
     '[disabled] .mdi-icon': {
       fill: theme.palette.primary.light,
     },
-    a: {
-      color: theme.palette.text.primary,
+   a: {
+      color: #FFF,
```

Comment:
I think you need to wrap it as a string, something like `'#FFF'` to add CSS in JS